src/components/FilterBarInline/FilterBarInline.tsx

```diff
@@ -0,0 +1,142 @@
+import { MultiSelectWidths, SelectedItems } from "../MultiSelect/MultiSelect";
+import useNYPLBreapoints from "../../hooks/useNYPLBreakpoints";
+import ComponentWrapper from "../ComponentWrapper/ComponentWrapper";
+
+export type FilterBarItemsType =
+  | (boolean | number | number[] | string | string[] | SelectedItems)[];
+
+interface FilterBarInlineProps {
+  /** The className of the FilterBarInline. */
+  className?: string;
+  /** ID that other components can cross reference for accessibility purposes. */
+  id?: string;
+  /** Optional string value used to set the text for a `Heading` component, or
+   * a DS Heading component that can be passed in.
+   */
+  heading?: string | JSX.Element;
+  /** Renders the layout of the FilterBarInline child components in a row or column. */
+  layout?: LayoutTypes;
+  /** Function for the global `Clear Filters` button. If passed the `Clear Filters` button will render. */
+  onClear?: () => void;
+  /** Function for the global `Apply Filters` button. If passed the `Apply Filters` button will render. */
+  onSubmit?: () => void;
+  /** Render prop function used to render child components with updated props. */
+  renderChildren: ({
+    isBlockElement,
+    layout,
+    width,
+  }: {
+    isBlockElement?: boolean;
+    layout?: LayoutTypes;
+    width?: MultiSelectWidths;
+  }) => React.ReactNode;
+  /** The selected items state (items that were checked by user). */
+  selectedItems?: FilterBarItemsType;
+}
+
+/**
+ * `FilterBarInline` is a wrapper component for filter components and individual form input components.
+ * The DS considers the following components as filter components: `CheckboxGroup`, `MultiSelectGroup`,
+ * `RadioGroup`, `Checkbox`, `DatePicker`, `MultiSelect`, `Radio`, `Select`, `Slider`, `TextInput`, and `Toggle`.
+ *
+ * The wrapped components/component groups can be displayed in a column or
+ * in a row layout. `FilterBarInline` can render additional `Clear All` and a `Apply Filters` buttons. The two
+ * _optional_ buttons are controlled by the `onClear` or `onSubmit` props respectively.
+ */
+
+export const FilterBarInline: ChakraComponent<
+  React.ForwardRefExoticComponent<
+    React.PropsWithChildren<FilterBarInlineProps> &
+      React.RefAttributes<HTMLDivElement>
+  >,
+  FilterBarInlineProps
+> = chakra(
+  forwardRef<HTMLDivElement, React.PropsWithChildren<FilterBarInlineProps>>(
+    (props, ref?) => {
+      const {
+        children,
+        className,
+        id,
+        heading,
+        layout = "row",
+        onClear,
+        onSubmit,
+        selectedItems,
+        renderChildren,
+        ...rest
+      } = props;
+
+      const { isLargerThanMobile } = useNYPLBreapoints();
+      const finalWidth = isLargerThanMobile
+        ? layout === "row"
+          ? "fitContent"
+          : "full"
+        : "full";
+
+      return (
+        <ComponentWrapper
+          className={className}
+          id={`filter-bar-inline-${id}`}
+          headingText={heading}
```

Comment:
Similar to what is done in `Banner`, the `Heading` element needs to be set to default size.  In this case, it should be set to `heading5`.  To go along with that, add a new `FilterBarInline Heading` section right after the `Accessibility` section.  The new section should more or less mirror what we have in the `Banner` docs.

https://nypl.github.io/nypl-design-system/reservoir/v3/?path=/docs/components-feedback-banner--docs#banner-heading